modules/fedimint-ln-common/src/lib.rs

```diff
@@ -343,6 +343,8 @@ pub struct LightningGateway {
     pub gateway_redeem_key: secp256k1::PublicKey,
     pub node_pub_key: secp256k1::PublicKey,
     pub lightning_alias: String,
+    /// URL to the gateway's versioned public API
+    /// (e.g. <https://gateway.example.com/v1>)
     pub api: SafeUrl,
```

Comment:
nit: rename this to `versioned_api` too

Reply:
> nit: rename this to `versioned_api` too

Idk, might be useful to keep this generic in case we need to pass non-versioned apis or non-RESTful api?